rules.ini

```
@@ -170,6 +170,7 @@ Intel_OID = (?:^|/)OpenImageDenoise\.dll$
 Intel_XeSS = (?:^|/)(?:ig|lib)xess\.dll$
 iZotope = (?:^|/)iZotope\.dll$
 JVM = (?:^|/)bin/java(?:\.exe|\.dll)?$
+Lua = (?:^|/)(?:lua50\.dll|lua51\.dll|lua52\.dll|lua53\.dll|lua54\.dll|lua5.0\.dll|lua5.1\.dll|lua5.2\.dll|lua5.3\.dll|lua5.4\.dll|liblua50\.so|liblua51\.so|liblua52\.so|liblua53\.so|liblua54\.so|\.lua)$
```

```suggestion
Lua[] = (?:^|/)(?:lib)?lua5.?[01234]\.(?:dll|so)$
Lua[] = \.lua$
```
untested so might be wrong
(run the scripts and fix things if you can)